x-pack/plugin/eql/qa/common/src/main/resources/test_missing_events.toml

```diff
@@ -0,0 +1,387 @@
+[[queries]]
+name = "missingInBetween_noMatches"
+query = '''
+  sequence with maxspan=1s
+    [ test1 where tag == "normal" ]
+   ![ test1 where tag == "nonexisting"]
+    [ test1 where tag == "normal" ]
+'''
+expected_event_ids = [1, -1, 2,
```

Comment:
Wdyt about changing the `-` sign of the results to `!` as well? To me, it feels a bit more intuitive when looking over the tests...

Reply:
Do you mean in the `expected_event_ids`? It's a `long[]`, that's why I used a negative value (I cannot even use null)

Reply:
Yeah, nevermind. I forgot about the pains and limitations on working with toml.